sdk.go

```diff
@@ -116,7 +116,7 @@ func Insecure(insecure bool) ServeOption {
 
 // Serve the supplied Function by creating a gRPC server and listening for
 // RunFunctionRequests. Blocks until the server returns an error.
-func Serve(fn v1beta1.FunctionRunnerServiceServer, o ...ServeOption) error {
+func Serve(fn v1beta1.FunctionRunnerServiceServer, maxMsgSize int, o ...ServeOption) error {
```

Comment:
Please make this a `ServeOption`, like network and address.

Reply:
Just did it!